weldx/transformations/types.py

```diff
@@ -1,25 +1,15 @@
 """shared type definitions."""
 from abc import abstractmethod
-from typing import TYPE_CHECKING, List, Protocol, Union, runtime_checkable
+from typing import List, Protocol, Union, runtime_checkable
 
 import numpy as np
-import pandas as pd
-import pint
 import xarray as xr
 from scipy.spatial.transform import Rotation
 
-if TYPE_CHECKING:  # pragma: no cover
-    import weldx
-
 types_coordinates = Union[xr.DataArray, np.ndarray, List]
 types_orientation = Union[xr.DataArray, np.ndarray, List[List], Rotation]
-# todo: remove next two definitions if possible
-types_timeindex = Union[pd.DatetimeIndex, pd.TimedeltaIndex, pint.Quantity]
-types_time_and_lcs = Union[
-    types_timeindex, "weldx.transformations.LocalCoordinateSystem"
-]
-
 
+# remove ???
 @runtime_checkable
 class SupportsTime(Protocol):
     """An ABC with one abstract method time."""
```

Comment:
Can this be removed? I found no usages and I guess it was supposed to do the same thing as `TimeDependent`

Reply:
I guess that is not needed any longer